source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+Regardless of your expertise, we recommend reading this section before starting your deployment.
+
+***********************
+Technical  Requirements
+***********************
+
+`Supported Mattermost Server Versions <https://github.com/mattermost/mattermost-mobile/blob/master/CHANGELOG.md>`_ 
+    Minimum requirements for the Mattermost Server is maintained in our documentation (link above). 
+    **However, we do recommend running the latest version** of the Mattermost Server as it contains the most recent features and any applicable security updates.
+
+    If this is not possible, we encourage you to bon at least version 5.12 of the Mattermost Server. This release has a number of critical feature updates that will ensure compatibility in a number of areas, including the Mattermost push notification service (MPNS).
+
+`Supported Devices/Mobile Device Requirements <https://docs.mattermost.com/install/requirements.html#mobile-apps>`_
+    Basic mobile device requirements are provided in the link above.
+
+`Network Requirements <https://>`_
+
+********************************
+Public App Store vs. Custom Apps
+********************************
+
+The most critical decision you will make in your mobile deployment will be whether to use the apps provided by Mattermost via the public App Stores (i.e. Google Play And Apple App Store), or to build and distribute custom versions of the Mattermost apps. Below we provide a very general overview of these options.
+
+Public App Store
+================
+
+Utilizing the public app stores greatly reduces the deployment time and is our recommended approach. Key benefits include… 
+
+- Easier deployment, driven by user needs.
+- Ability to use the Mattermost Push Notification Service, or MPNS.
+- Apps automatically updated with the latest features, enhancements and security updates.
+
+Again, this is the recommended route which is as simple as downloading the Mattermost mobile application from the corresponding store, entering a URL to point towards your Mattermost server, and logging in.
+
+Custom Apps
+===========
+
+If you desire to customize the applications, or do not want your users downloading the application from the public app stores, you will need to build the apps yourself. 
+
+As the apps are an open source project, it will require a fork, and your team will be responsible for maintaining the fork, as well as keeping the fork up to date with any changes made by Mattermost.
+
+This process can be complicated. It will also greatly increase deployment time, not only initially, but whenever the apps need to be updated. 
+
+We recommend having your development team take a look at the documentation to ensure they understand the scale and requirements of this path. In general, this route will present some challenges, including…
+
+- Obtaining/providing certificates for your custom application
+- Signing your custom applications
+- Distributing your applications via public or private app stores
```

Comment:
again public/private app stores is not a thing

Reply:
When using an EMM. There is the notion of an “Enterprise of private App Store.” Maybe this concept is conflated: https://developers.google.com/android/work/play/emm-api/private-apps. Cc @enahum

Reply:
The additional comment indicates what we are really trying to make a distinction with is "Official" (which is also public) and "Private" options which include Enterprise App Stores and an Emm App Catalog. I think getting behind a distinction like this would be important not only for this doc, and the documentation, but the overall language used internally by all staff. Does those terms meet the desired outcome?

My worry with Mattermost Distributed, is that I don't think it's clear. And to your point, on the Android side, it could potentially make someone think there is a "Mattermost App Store" somewhere.

Reply:
solved in meeting